src/label/styles/label.css

```diff
@@ -0,0 +1,11 @@
+/* TODO: this should go in a util.css */
```

Comment:
Should this go in `variables.css` in widgets/styles?
I think most things in the `variables.css` should be in a theme, and classes like this *do* belong in a x-theme variables file.